playbook/app/pb_kits/playbook/pb_timestamp/timestamp.rb

```diff
@@ -73,12 +82,38 @@ def format_updated_string
 
       def format_elapsed_string
         user_string = show_user ? " by #{text}" : ""
-        datetime_string = " #{time_ago_in_words(pb_date_time.convert_to_timestamp)} ago"
+        datetime_string = " #{pb_time_ago(pb_date_time.convert_to_timestamp)} ago"
+        datetime_string[1] = hide_updated ? datetime_string[1].upcase : datetime_string[1]
         updated_string = hide_updated ? "" : "Last updated"
-
         "#{updated_string}#{user_string}#{datetime_string}"
       end
 
+      def pb_time_ago(value)
+        time_ago = DateTime.now.to_i - value.to_i
+        case time_ago
+        when (0...SECS_PER_MIN)
```

Comment:
It makes sense to me to have 0-60 seconds for "seconds ago". But, this ends up being different from the React way: "0-44.99 seconds".